frontend/package.json

```diff
@@ -17,6 +17,8 @@
     "@fortawesome/free-solid-svg-icons": "^6.2.1",
     "@fortawesome/vue-fontawesome": "^3.0.2",
     "@popperjs/core": "^2.11.6",
+    "@vuelidate/core": "^2.0.0",
+    "@vuelidate/validators": "^2.0.0",
```

Comment:
Should update wiki https://github.com/bcgov/nr-forests-access-management/wiki/FAM-Frontend-Design to discuss choice of vuelidate versus other candidates like vee-validate.

Reply:
I had decided to go with vuelidate but I faced an issue with non-visible fields triggering validation no matter what. So I decided to switch to vee-validate because it seems to be more flexible, yet more complex. I will populate the Wiki with my findings.